sql/core/src/test/resources/sql-tests/results/pivot.sql.out

```diff
@@ -1,5 +1,5 @@
--- Automatically generated by SQLQueryTestSuite
--- Number of queries: 31
+-- Automatically generated by SparkServiceSQLQueryTestSuite
```

Comment:
`SparkServiceSQLQueryTestSuite` -> `SQLQueryTestSuite`?

Reply:
Good catch. Do you have any idea how it has turned out this way?